NuRadioReco/framework/sim_station.py

```diff
@@ -12,12 +12,10 @@
 
 class SimStation(NuRadioReco.framework.base_station.BaseStation):
 
-    def __init__(self, station_id, sampling_rate=None, trace=None):
+    def __init__(self, station_id):
         NuRadioReco.framework.base_station.BaseStation.__init__(self, station_id)
         self.__magnetic_field_vector = None
         self.__simulation_weight = None
-        if(trace is not None and sampling_rate is not None):
```

Comment:
This is removed because we now don't store the electric field directly in the simStation anymore, right? Just checking.

Reply:
Yes, we must have missed this part when we introduced the dedicated E-Field class